Spigot-Server-Patches/0330-Avoid-item-merge-if-stack-size-above-max-stack-size.patch

```diff
@@ -0,0 +1,25 @@
+From 311e9503ab54d720ccf3304c144ea85fb9e4ca93 Mon Sep 17 00:00:00 2001
+From: Hugo Manrique <hugmanrique@gmail.com>
```

Comment:
Note: This is not the same email you added to the MIT license file. You probably want to add this mail to to `LICENSE.md` or reconfigure Git to use the other mail address.